dragonchain/broadcast_processor/broadcast_functions.py

```diff
@@ -102,13 +105,30 @@ def increment_storage_error_sync(block_id: str, current_level: int) -> None:
 
 async def get_blocks_to_process_for_broadcast_async() -> List[Tuple[str, int]]:
     """Get the blocks scheduled to be checked by the broadcast processor right now
-    Args:
     Returns:
         List of (block_id, score) tuples to be processed by the broadcast processor (limit of 1000)
     """
     return await redis.z_range_by_score_async(IN_FLIGHT_KEY, 0, int(time.time()), withscores=True, offset=0, count=1000)
 
 
+async def get_notification_verifications_for_broadcast_async() -> set:
+    """Get the notifications scheduled to be checked by the broadcast processor right now
+    Returns:
+        List of notifications to be re-transmitted by the broadcast processor (limit of 1000)
```

Comment:
incorrect limit. Also this method probably needs a decode since it's getting data out of redis. Also I'm not 100% sure this returns a set. I think the other redis library returns a list for smembers, and we have to coerce it. I would double check and confirm.

Reply:
checked. Confirmed. Is a set